app/controllers/admin/dashboard_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
       @trending_hashtags     = TrendingTags.get(7)
       @profile_directory     = Setting.profile_directory
       @timeline_preview      = Setting.timeline_preview
+      @keybase_integration   = Setting.enable_keybase
```

Comment:
Technically you are disabling the entire identity proofs feature, not just Keybase. I suggest renaming all variables accordingly.

Reply:
Well, I did really intend to disable keybase itself, not the identity proofs feature which currently only implement keybase. I considered checking whether the provider was “keybase” in the proof creation thing but figured we would change it if/whenever we add other identity proof providers.

Reply:
I mean, I think it makes sense for the option to be specifically about keybase, as people might not have the same opposition towards other hypothetical identity proof providers.
Would you want me to explicitly check for the `keybase` provider in the code? Or change the option anyway to control the whole feature rather than keybase itself?